doc/releases/changelog-0.8.1.md

```diff
@@ -2,14 +2,59 @@
 
 <h3>New features</h3>
 
-<h3>Improvements</h3>
+* Zero-Noise Extrapolation (ZNE) local folding: Introduces the option to fold gates locally as well
```

Comment:
@josh146 Can you take a look at the changelog?

Reply:
Will do!

Reply:
Done